exporter/azuremonitorexporter/AUTHENTICATION.md

```diff
@@ -20,6 +20,18 @@ exporters:
       connection_string: ${env:APPLICATIONINSIGHTS_CONNECTION_STRING}
 ```
 
+## Authenticator Extension
+
+You can use any standard OTEL authenticator extension to add authentication to outgoing requests.
```

Comment:
Should only support `azureauth` extension?

Reply:
Technically it could work with any standard otel authenticator support, as long as the token generated by the extension is accepted by Microsoft. I put that in case some people want to use custom extensions for specific use-cases. The example is with `azureauth` extension so I think it's clear enough for users what the typical usage would be.